### ISSUE TITLE
 forgot to define html5_data attribute column x error

### ISSUE THREAD

**Issue by Organizer** [vedmack/yadcf#149]
I currently get a "It seems you have forgot to define html5_data attribute for column x" error message when any field/row in my serverside ajax/JSON returns null" as value. Any logic or suggestion to this, tested 0.8.6 and latest beta + latest datatables version. 

Some related columns simply defined by -> "data": "6", "visible": false, "searchable": false and yadcf {    column_number: 6, filter_type: "autocomplete"  }

**Comment by vedmack** [comment]
Hi,
You can't filter a column that is set to `"searchable": false` (not with yadcf and not with datatbles)

If you'll have any similar issues with a column that is not set to `"searchable": false` feel free to reopen this issue and provide additional information

**Comment by charliechihuahua** [comment]
Hi,
I get the same error. It occurs when a field in the Json Data array is null and the Data Array is no associative array. Tested in beta 0.8.7

I think the problem is in the function parseTableColumn line number 1487. 
```javascript
if (typeof col_inner_data === 'object') {
	if (columnObj.html5_data !== undefined) {
		col_inner_data = col_inner_data['@' + columnObj.html5_data];
	} else {
		alert('Looks like you have forgot to define the html5_data attribute for the ' + columnObj.column_number + ' column');
		return;
	}
}
```
if the Field Value `data[j]._aData[columnObj.column_number]` is null it is an object.
`if (col_inner_data !== null && typeof col_inner_data === 'object')` solved the problem.

**Comment by SoerenWeber** [comment]
I just experienced the same behavior @charliechihuahua observed and was able to solve it by using their solution with version 0.9.4.beta.35.

Do you see a reason a drawback in this solution, @vedmack ? I could open a PR if you like. :)

**Comment by vedmack** [comment]
@SoerenWeber , yes that would be great

**Comment by vedmack** [comment]
committed the fix in 0.9.4.beta.36
